src/main/java/pages/LoginPage.java

```diff
@@ -113,5 +126,101 @@ public String getSignInButtonFontColorAsHex() {
+        return driver.findElement(newCustomerContainer).isDisplayed();
+    }
+
+    public boolean isTryDemoCustomerAccessSectionDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessSection).isDisplayed();
+    }
+
+    public boolean isNewCustomersTitleDisplayed(){
+        return driver.findElement(newCustomersTitle).isDisplayed();
+    }
+
+    public String getNewCustomersTitleText() {
+        return driver.findElement(newCustomersTitle).getText();
+    }
+
+    public boolean isNewCustomersMainTextDisplayed(){
+        return driver.findElement(newCustomersMainText).isDisplayed();
+    }
+
+    public String getNewCustomersMainText() {
+        return driver.findElement(newCustomersMainText).getText();
+    }
+
+    public boolean isCreateAccountButtonEnabled(){
+        return driver.findElement(createAccountButton).isDisplayed();
+    }
+
+    public String getCreateAccountButtonText() {
+        return driver.findElement(createAccountButton).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessTitleDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessTitle).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessTitleText() {
+        return driver.findElement(tryDemoCustomerAccessTitle).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessEmailDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessEmail).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessEmailText() {
+        return driver.findElement(tryDemoCustomerAccessEmail).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessPasswordDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessPassword).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessPasswordText(){
+        return driver.findElement(tryDemoCustomerAccessPassword).getText();
+    }
+
+    public String getCreateAccountButtonFontColorAsHex() {
+        WebElement createAccountButtonElement = driver.findElement(createAccountButton);
+        String createAccountButtonColorFontRgba = createAccountButtonElement.getCssValue("color");
+        return Color.fromString(createAccountButtonColorFontRgba).asHex();
+    }
+
+    public String getCreateAccountButtonBackgroundColorAsHex(){
+        WebElement createAccountButtonElement = driver.findElement(createAccountButton);
+        waitUntilElementContainsText(driver,createAccountButton,"Create an Account");
+        String createAccountButtonColorBackgroundRgba = createAccountButtonElement.getCssValue("background");
+        return Color.fromString(createAccountButtonColorBackgroundRgba).asHex();
+    }
+
+    public String getTryDemoCustomerAccessSectionBackgroundColorAsHex(){
+        WebElement tryDemoCustomerAccessSectionElement = driver.findElement(tryDemoCustomerAccessSection);
+        String tryDemoCustomerAccessSectionBackgroundRgba = tryDemoCustomerAccessSectionElement.getCssValue("background");
```

Comment:
Here is the same issue as for case with the background for Create Account button.